src/components/search-form/search-form.js

```diff
@@ -229,6 +258,77 @@ class SearchForm extends Component {
       );
   }
 
+  renderAccessTypesFilter() {
+    const {
+      accessTypesStoreData,
+      searchByAccessTypesEnabled,
+      searchFilter = {},
+    } = this.props;
+
+    const {
+      'access-type': accessTypes = ''
+    } = searchFilter;
+
+    const {
+      sections,
+    } = this.state;
+
+    const accessTypesList = accessTypes ? accessTypes.split(',').map(accessType => {
+      return accessType.toLowerCase();
+    }) : [];
```

Comment:
You can remove `return` from here
```suggestion
    const accessTypesList = accessTypes?.split(',').map(accessType => accessType.toLowerCase()) || [];
```